src/components/ClusterManager.js

```diff
@@ -478,6 +478,7 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     const currentCluster = this.getCurrentCluster()
     const currentStatus = currentCluster && currentCluster.status
     const running = currentStatus === 'Running'
+    const spendingClusters = _.remove({ status: 'Deleting' }, _.remove({ status: 'Error' }, clusters))
```

Comment:
```suggestion
    const spendingClusters = _.remove(({ status }) => _.includes(status, ['Deleting', 'Error']), clusters)
```
A teeny bit longer, but clearer, I think.